include/ttmlir/Target/Common/types.fbs

```diff
@@ -186,3 +187,8 @@ enum MathFidelity : uint8 {
     HiFi3         = 3,
     HiFi4         = 4,
 }
+
+enum UnpackToDestMode : uint8 {
+    UnpackToDestFp32 = 0,
+    Default = 1,
+}
```

Comment:
Should we move this to `program.fbs` since it's specific to programs?

Reply:
Sure, good idea. Done (fixup, force push), this changed namespaces from:
```
tt::target::UnpackToDestMode::UnpackToDestFp32
```
to:

```
tt::target::metal::UnpackToDestMode::UnpackToDestFp32
```